src/vs/base/common/htmlContent.ts

```diff
@@ -19,6 +19,8 @@ export interface IMarkdownString {
 	readonly isTrusted?: boolean | MarkdownStringTrustedOptions;
 	readonly supportThemeIcons?: boolean;
 	readonly supportHtml?: boolean;
+	/** @internal */
+	readonly supportAlertSyntax?: boolean;
```

Comment:
I don't think we really use `@internal`, but @mjbvz can decide if he wants this or not.

Reply:
Actually, what's the `@internal` for?